tests/scripts/mbedtls_test.py

```diff
@@ -112,8 +110,8 @@ def __parse(self, data_f):
             args = parts[1:]
             args_count = len(args)
             if args_count % 2 != 0:
-                raise TestDataParserError("Number of test arguments should "
-                                          "be even: %s" % line)
+                raise TestDataParserError(("Number of test arguments({}) should "
+                                          "be even: {}").format( args_count, line ) )
```

Comment:
Does this pass `check-python-files.sh`? I'm curious because the spaces around the `(` are Mbed TLS style and not Python style.

Reply:
Since it's not part of the CI, or part of the pre-push script it wasn't run... but the python itself doesn't crash
The script fails on a pylint error:
`pylint: error: no such option: -j`

Reply:
The pylint version I am using is 1.1.0

Reply:
It's part of the CI that gatekeepers have to run manually, also known as `all.sh`. :)

I have `-j` on my version, which is pylint 1.8.3. It just means "run multiple jobs in parallel" and you could omit it. It's present just to speed up all the checks.

Reply:
my `all.sh` never failed. and I had to install pylint only now

After removing the `-j` in the script, I still get pylint errors, investigating at the moment

Reply:
Unfortunately, `all.sh` checks for the existence of pylint, and only if it is present does it do the checks. I think that's a bug, as it means we might think we tested for something when we didn't.

Reply:
I create #2252 to run the script when pushing a PR